### ISSUE TITLE
Keyboard Issue: Integrating with WKWebview

### ISSUE THREAD

**Issue by woniesong92** [meteoric/meteor-ionic#272]
I saw all the threads in #130 and I am still having troubles with the keyboard behavior on my app. I would really appreciate it if you could take a look at it (iOS).

Here's what's happening:
https://www.youtube.com/watch?v=McBuwu5Ef78

I forked Meteoric and modified [ionKeyboard.js](https://github.com/woniesong92/meteor-ionic/blob/master/components/ionKeyboard/ionKeyboard.js) file. Basically, I removed all the scrolling behavior except for adding `bottom:` to keyboard-attached elements and scrolling to top whenever a keyboard appears. As you can see in the video, the behavior (scrolling down and up for the first 200ms) is not as desired although the final position of keyboard is okay.

Do you have an idea how I can resolve this issue? This is the last block for my app to be published and I would really like to think there is an answer. Thank you so much!

**Comment by woniesong92** [comment]
@nickw quick question: Does `IonKeyboard.disableScroll()` not work on WKWebkit??

**Comment by woniesong92** [comment]
Tl;dr: `cordova.plugins.Keyboard.disableScroll(true);` does not work. When the input field is at the bottom, the entire screen will be scrolled down. Is there anyway to prevent this from happening?

**Comment by insaneinc01** [comment]
Hello @woniesong92, were you able to get this working?

**Comment by nickw** [comment]
I haven't tried yet. I'm wondering if the upgraded Cordova in Meteor 1.2 (`meteor update --release METEOR@1.2-rc.7`) will fix this?

**Comment by jorjordandan** [comment]
I think the issue probably comes from the fact that you're using a not particularly regularly maintained meteor wrapper for the telerik plugin instead of regular cordova. I can't wait to be able to use WKWebviews for meteoric either, but there are probably a few wrinkles to be ironed out in both those repos. I even tried to write my own WKWebview wrapper, but got stuck at some authentication issues.... Good luck!
